### DISCUSSION TITLE
Smartmap Erweiterung

### DISCUSSION THREAD

**Discussion by thomaskofler** [e-netze/webgis-community#214] [General]
Hallo Jürgen,

wie besprochen, wir hätten gerne eine Erweiterung bei der Smartmap und zwar, dass man wie bei Google-Maps einen Switcher zur Verfügung hat, wo man zwei Hintergrundkarten schalten kann (zB Basemap/Orthofoto).

Umsetzung wäre ideal bis Ende des Jahres.

LG Thomas

**Comment by gis-eni** [comment]
Erweiterung wurde umgesetzt:

Sind mehere Basemaps in einer Smartmap eingebunden, erscheint links unten in der Karte ein Button ``Hintergrund``:

![image](https://github.com/user-attachments/assets/3a46af97-d2a2-41ea-81b9-5ffa56f37a34)

Klickt man auf den Button, werden stehen alle Basemaps per Button zur Auswahl. Der Button mit der aktuellen Basemap wird hervorgehoben:

![image](https://github.com/user-attachments/assets/3e10e0e6-27d1-4353-a325-a83b2780b8b5)

Klickt man auf einen Button, ändert sich der Karten Hintergrund und die Button Leiste wird wieder zusammen geklappt.

**Comment by thomaskofler** [comment]
Super, danke für die schnelle Umsetzung!! Ab welcher Version ist dann die Funktion verfügbar?
